accord-core/src/test/java/accord/burn/BurnTest.java

```diff
@@ -229,6 +233,7 @@ static void burn(Random random, TopologyFactory topologyFactory, List<Id> client
     public static void main(String[] args) throws Exception
     {
         Long overrideSeed = null;
+//        Long overrideSeed = -7320078316311161123L;
```

Comment:
leftover seed

Reply:
I leave some seed commented out to make it easier to switch. Not sure if it matters much at this point in development, but perhaps I should make it cleaner going forwards.